config/cmake/modules/FindAxom.cmake

```diff
@@ -18,4 +18,13 @@ include(MfemCmakeUtilities)
 # Note: components are enabled based on the find_package() parameters.
 mfem_find_package(Axom AXOM AXOM_DIR "include" "" "lib" ""
   "Paths to headers required by Axom." "Libraries required by Axom."
-  ADD_COMPONENT Axom "include" axom/config.hpp "lib" axom)
```

Comment:
Can you just comment on why this changed from specifying Axom component to specifying the individual components underneath Axom?

Reply:
I couldn't get the Axom component working without making the change.  I didn't investigate too much, but the include directories and libraries were empty.